docs/release/trg-0/trg-1-08.md

```diff
@@ -0,0 +1,31 @@
+---
+title: TRG 1.08 - Interfaces Documentation
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status     | Created      | Post-History          |
+|------------|--------------|-----------------------|
+| Draft      | 31-Jan-2024  | created initial Draft |
+
+## Why
+
+Interfaces Documentation is crucial for integration testing and ensuring seamless interoperability for Developers consuming the product API
+
+## Description
+
+- The interface documentation **has to** be stored in the folder: ```docs/interfaces/```
+- API documentation contains all relevant interfaces for integration testing and is completed for the relevant product version.
+- The context of the request **has to** be clear (sequence of API calls)
+- Link to the API-Swagger-File is available.
+- Functional and non-functional commitment of the interface.
+
+## Best Practice
```

Comment:
Again, remove the process engagement parts

Reply:
yes, delete current lines 28 to end.
new text under best practice (bulletpoints):
adhere to standards like Swagger or openAPI
consider functional and non-functional comittments of each interface
context of a request must be clear (sequence of API calls)